asserts/snap_asserts.go

```diff
@@ -171,16 +172,16 @@ func (snaprev *SnapRevision) SnapDigest() string {
 	return snaprev.Header("snap-digest")
 }
 
+// SnapSize returns the size in bytes of the snap submitted to the store.
+func (snaprev *SnapRevision) SnapSize() uint64 {
+	return snaprev.snapSize
+}
+
 // SnapRevision returns the revision of the snap-id assigned to this build.
```

Comment:
Fixed. Also updated some doc comments that suggested a coupling with snap-build.